src/main/scala/diplomacy/Nodes.scala

```diff
@@ -1235,7 +1235,8 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
       data   = bundleIn(i))
   }
 
-  private var bundlesSafeNow = false
+  private[diplomacy] var bundlesSafeNow = false
```

Comment:
Having to soften this guard is a symptom of something being wrong with the existing API, IMO. E.g. if we just had `instantiated` would you `Select` object even have to be a part of this library? (Not saying it shouldn't be anyways, just that it wouldn't have to be)

Reply:
yeah `Select` could be a completely separate project, but it would require a public accessor for `instantiated`.

Reply:
Hmm, I agree as written it would since you put a `require` on `instantiated` directly, but thinking about it more, is that require in your user code even necessary? Put another way, if we retire `bundlesSafeNow` and replace it with `instantiated` isn't there already some internal requirement on `bundlesSafeNow` that trips if you access the bundle? I am guessing you hit that yourself, since you went in and toggled it off. So if that internal requirement gets changes to be on `instantiated`, do you select functions even need their own `require` anymore?

Reply:
yeah you're right